build/Publish.targets

```diff
@@ -175,15 +175,23 @@
                     AccountKey="$(PublishBlobFeedKey)"
                     ItemsToPush="@(PackagesToPublishToTransport)"
                     Overwrite="$(PushToBlobFeed_Overwrite)"
+                    ManifestBuildId="$(BuildNumber)"
+                    ManifestCommit="$(CommitHash)"
+                    ManifestName="https://github.com/aspnet/Home"
```

Comment:
I think the `/`s from the url will make this not work--it's used to name a blob and I haven't plumbed through escaping. When I said repo name in [publish.md](https://github.com/dotnet/core-eng/blob/master/Documentation/Orchestrated-Build/Api/publish.md#cookbook-using-pushtoblobfeed-to-generate-a-manifest) I only meant the last bit, so `Home` in this case... which isn't very descriptive. I didn't consider repos that primarily use the organization to distinguish themselves, sorry.

I think `aspnet-Home` would be reasonable, but `aspnet-Universe` might be best. The `CommitHash` is for Universe, right?

Reply:
We'll use `aspnet` for now. `CommitHash` points to the current repo viz Universe.